test/ng/filter/limitToSpec.js

```diff
@@ -71,6 +105,11 @@ describe('Filter: limitTo', function() {
     expect(limitTo(items, 'undefined')).toEqual(items);
     expect(limitTo(items, null)).toEqual(items);
     expect(limitTo(items, undefined)).toEqual(items);
+    expect(limitTo(arrayLike, 'bogus')).toEqual(arrayLike);
+    expect(limitTo(arrayLike, 'null')).toEqual(arrayLike);
+    expect(limitTo(arrayLike, 'undefined')).toEqual(arrayLike);
+    expect(limitTo(arrayLike, null)).toEqual(arrayLike);
+    expect(limitTo(arrayLike, undefined)).toEqual(arrayLike);
```

Comment:
I would rather use `toBe` in this test as it captures the intent better. (I know it was `toEqual` even before this PR.)